sphinx/pycode/parser.py

```diff
@@ -479,7 +479,7 @@ def parse(self) -> None:
 
     def parse_comments(self) -> None:
         """Parse the code and pick up comments."""
-        tree = ast.parse(self.code.encode())
+        tree = ast.parse(self.code)
```

Comment:
Note:
The reference of `ast.parse()` says:
>Parse the source into an AST node. Equivalent to compile(source, filename, mode, ast.PyCF_ONLY_AST).

And `compile()`'s says:
>source can either be a normal string, a byte string, or an AST object.

This change calls `ast.parse()` correctly. So +1 for merging without tests.

Reply:
Thank you for patience to such badly-skilled contributor as me :-)